### ISSUE TITLE
PATH addition seems incorrect(?)

### ISSUE THREAD

**Issue by marcdw1289** [NsCDE/NsCDE#80]
I just updated OpenBSD to 7.0, immediately followed by recompiling fvwm3, then updating to NsCDE 1.3. All good.
Today I just noticed that added to my PATH is /usr/local/NsCDE.
Is that correct? Seems odd given that there are no executables there. From where is that path set?

On a side note, the desktop takes a lot longer to startup. Probably an OpenBSD 7 thing.

**Comment by NsCDE** [comment]
Hi

It depends where you have installed NsCDE 1.3, but it doesn't have to be in $PATH. I don't know which mechanism add this to your path. I have also upgraded my OpenBSD VM where I test compatibiity, and that didn't happened as I can see now.

Longer start? Who knows ... try to start X session as xterm from ~/.xsession, and then run $NSCDE_ROOT/bin/nscde from there, and watch where and why it hangs. Again, I didn't noticed any slowdown in my OpenBSD VM.

BTW, in a month, there will be 3 years of NsCDE, and I will release 2.0 which will have normal "./configure; make; make install" procedure, it can be put anywhere, and will be much more FHS and packager/port friendly.

**Comment by marcdw1289** [comment]
Sorry for the late response. The slightly longer delay before the desktop starts could be an OS thing. The only errors I had in my xsession log was from nscde_palette_colorgen.py. NsCDE was installed under /usr/local but it was trying and failing to find my selected palette under /opt.

I decided to start fresh and put NsCDE under /opt. /usr/local/NsCDE stopped showing up in my path after that. Still don't know why that was happening.

Side note. If you're not on OpenBSD 7 take note that the qt5-style-plugins has been removed from ports and leaving it installed results in non-working apps. Though I have qt5ct installed and the QT_QPA_PLATFORMTHEME=qt5ct variable set, my Qt apps don't match the rest. Will check on that when I have the time.

NsCDE is my desktop on three OSes now. Lovin' it. Thanks for this.
